lib/rouge/lexers/actionscript.rb

```diff
@@ -39,38 +39,38 @@ class Actionscript < RegexLexer
 
         rule %r([^/]\n), Error, :pop!
 
-        rule /\n/, Error, :pop!
-        rule /\[\^/, Str::Escape, :regex_group
-        rule /\[/, Str::Escape, :regex_group
-        rule /\\./, Str::Escape
+        rule %r/\n/, Error, :pop!
+        rule %r/\[\^/, Str::Escape, :regex_group
+        rule %r/\[/, Str::Escape, :regex_group
+        rule %r/\\./, Str::Escape
         rule %r{[(][?][:=<!]}, Str::Escape
-        rule /[{][\d,]+[}]/, Str::Escape
-        rule /[()?]/, Str::Escape
-        rule /./, Str::Regex
+        rule %r/[{][\d,]+[}]/, Str::Escape
+        rule %r/[()?]/, Str::Escape
+        rule %r/./, Str::Regex
       end
 
       state :regex_end do
-        rule /[gim]+/, Str::Regex, :pop!
+        rule %r/[gim]+/, Str::Regex, :pop!
         rule(//) { pop! }
       end
 
       state :regex_group do
         # specially highlight / in a group to indicate that it doesn't
         # close the regex
-        rule /\//, Str::Escape
+        rule %r/\//, Str::Escape
```

Comment:
Using `%r` syntax has one other advantage: One could adjust the delimiter to reduce escaping in the pattern to improve readability:
```diff
-  %r/\//
+ %r(/)
```

Reply:
This is true. I updated all the lexers using a simple regex, though, and it was easier to leave them as they were rather than go through each one manually.

I'd like to move towards a grand unified refactoring of lexers but am putting that off for the time being. Partly that's because I'm still not sure in my own head what the standards should be for lexers and partly because working through the outstanding PRs seems generally a better use of time (although I do get distracted by [things](https://github.com/ruby/strscan/pull/6) on occasion).

If you'd like to clean them up, you're welcome to submit a PR to my branch!

Reply:
Done!